lib/identity_code/estonia.rb

```diff
@@ -37,7 +37,7 @@ def check_digit
 
       if checknum == 10
         scales = [3, 4, 5, 6, 7, 8, 9, 1, 2, 3]
-        checknum  = combine_scale_with_number(scales, identity_code)
+        checknum = combine_scale_with_number(scales, identity_code)
```

Comment:
Can we keep reformatting-related changes separately? Otherwise this change seems to have nothing to do with "Fix test coverage for Estonia personal ID codes".

Reply:
Alright. Opening a new PR just to fix one space feels hilarious to me, to be honest. I'll add the space back.

Reply:
I meant separate commit, actually :)

Reply:
Something that would explain the change, like "Reformat" (this is what I use, but feel free to choose whatever you think is clear for the widest audience)

Reply:
The point is that if you see such commit message, you know right away that no logic is changed there, and you don't spend time digging into the details.